hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOzoneManagerPrepare.java

```diff
@@ -266,61 +233,23 @@ private void writeTestData(ObjectStore store, String volumeName,
     keyStream.close();
   }
 
-  private OMRequest buildPrepareRequest() {
-    PrepareRequest requestProto = PrepareRequest.newBuilder().build();
-
-    return OMRequest.newBuilder()
-        .setPrepareRequest(requestProto)
-        .setCmdType(Type.Prepare)
-        .setClientId(UUID.randomUUID().toString())
-        .build();
-  }
-
   private void waitAndCheckPrepared(OzoneManager om,
       long prepareRequestLogIndex) throws Exception {
     // Log files are deleted after the snapshot is taken,
     // So once log files have been deleted, OM should be prepared.
     LambdaTestUtils.await(timeoutMillis, 1000,
         () -> !logFilesPresentInRatisPeer(om));
-    checkPrepared(om, prepareRequestLogIndex);
+    Assert.assertTrue(checkPrepared(om, prepareRequestLogIndex));
   }
 
-  private void checkPrepared(OzoneManager om, long prepareRequestLogIndex)
+  private boolean checkPrepared(OzoneManager om, long prepareRequestLogIndex)
```

Comment:
Do we still need this method? Since we are no longer immediately checking the leader and waiting on all 3 OMs, can we just put these lines in `waitAndCheckPrepared`? This method on its own is kind of misleading, since it no longer checks that the logs have been removed, and therefore doesn't do a full check for preparedness.

Reply:
Due to my last comment, this still has one usage.